.cirrus.yml

```diff
@@ -301,7 +301,7 @@ report_stagetwo_task:
             dnf -y install python3-pexpect
         fi
     main_script: &stagetwo_test |
-        PYTHONPATH=tests/ avocado run -p TESTLOCAL=true --max-parallel-tasks=1 -t stagetwo tests/{cleaner,collect,report,vendor}_tests
+        PYTHONPATH=tests/:. avocado run -p TESTLOCAL=true --max-parallel-tasks=1 -t stagetwo tests/{cleaner,collect,report,vendor}_tests
```

Comment:
Why this change is required, please?

Reply:
We need `sos` to get the `shell_out` function. It was one of the errors in a previous push